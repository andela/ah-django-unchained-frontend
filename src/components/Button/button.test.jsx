import React from "react";
import { shallow } from "enzyme";
import Button from "./";

describe("Button", () => {
  it("should be defined", () => {
    expect(Button).toBeDefined();
  });
  it("should render correctly", () => {
    const tree = shallow(
      <Button
        type="submit"
        className="col-sm-6 btn btn-success form-control"
        text="login"
      />
    );
    expect(tree).toMatchSnapshot();
  });
});
