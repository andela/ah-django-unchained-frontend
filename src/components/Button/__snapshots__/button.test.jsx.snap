// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render correctly 1`] = `
<button
  className="col-sm-6 btn btn-success form-control"
  type="submit"
>
  login
</button>
`;

exports[`InputBox should render correctly 1`] = `
<button
  className="col-sm-6 btn btn-success form-control"
  type="submit"
>
  login
</button>
`;
